src/components/BandeauDemo.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+  <div class="bandeau-demo-wrapper" v-if="afficheBandeau">
+    <a class="bandeau-demo" target="_blank" rel="noopener" :href="link">
+      Démo
+    </a>
+  </div>
+</template>
+
+<script>
+export default {
+  name: "BandeauDemo",
+  computed: {
+    afficheBandeau() {
+      return process.env.VUE_APP_DEMO === "true"
```

Comment:
C'est surprenant comme condition. Je m'attendrais à un `process.env.VUE_APP_DEMO === true` ou `process.env.VUE_APP_DEMO` utilisé comme un _truthy_ mais j'ai l'impression que quelque chose m'échappe.

Reply:
VUE_APP_DEMO est une string c'est pourquoi j'ai du changer cette condition pour checker `VUE_APP_DEMO === "true" // string true`

Reply:
`The process.env object forces all of its properties to be of type string`